components/accept-financial-contributions/AcceptContributionsMyself.js

```diff
@@ -240,18 +243,8 @@ const bankAccountMutation = gqlV2`
 `;
 
 const addBankAccountMutation = graphql(bankAccountMutation, {
-  options: {
-    context: API_V2_CONTEXT,
-  },
-  props: ({ mutate }) => ({
-    addBankAccount: async ({ account, key, value }) => {
-      return await mutate({
-        variables: { account, key, value },
-        awaitRefetchQueries: true,
-        refetchQueries: [{ query: getLoggedInUserQuery }],
-      });
-    },
-  }),
+  name: 'addBankAccount',
```

Comment:
Use name instead of re-creating the function, we will just have to make sure to pass `variables`.